metadata-extractor-cve-2019-14262/README.md

```diff
@@ -0,0 +1,53 @@
+# Metadata Extractor CVE-2019-14262 Example
+
+This target replicates finding [CVE-2019-14262](https://nvd.nist.gov/vuln/detail/CVE-2019-14262), a stack exhaustion bug caused by uncontrolled recursion in version 2.1.0 of the metadata-extractor library ([CVSS Score](https://nvd.nist.gov/vuln-metrics/cvss): 7.5).
+
+This vulnerability was reported to the maintainers and resolved [here](https://github.com/drewnoakes/metadata-extractor/issues/419).
+
+Note that the CVE is for the C# version of the same code, where one cannot typically recover from a stack overflow.
+This example reproduces the vulnerability in the Java version to demonstrate the Java fuzzing capabilities of Mayhem.
```

Comment:
Might mention that we disable catching the stack overflow exception in the java patch.